### PR TITLE
Switch color change, removed header content border, reduced header sp…

### PR DESCRIPTION
**Pull request by kevinylee1991**
…acing

Made a few style changes madhura requested

### CODE REVIEW

File: src/app/plugins/kubernetes/components/infrastructure/InfrastructurePage.js [hunk 1]
```diff
@@ -19,12 +19,13 @@ const useStyles = makeStyles(theme => ({
...
     minHeight: 50,
   },
+
 }))
 
 const StatsToggle = ({ statsVisible, toggleStats }) => {
   return <FormControlLabel
     control={
-      <Switch onChange={toggleStats} checked={statsVisible} />
+      <Switch onChange={toggleStats} checked={statsVisible} color='primary' />
```
Comment (brennancheung) [inline src/app/plugins/kubernetes/components/infrastructure/InfrastructurePage.js]: double quotes for JSX attributes.

File: src/app/core/containers/AuthenticatedContainer.js [hunk 2]
```diff
@@ -52,7 +52,7 @@ const useStyles = makeStyles(theme => ({
     alignItems: 'center',
     justifyContent: 'flex-end',
     padding: '0 8px',
-    ...theme.mixins.toolbar,
+    minHeight: '48px',
```
Comment (brennancheung) [inline src/app/core/containers/AuthenticatedContainer.js]: Specify from `theme.spacing`.  I believe `1` unit in `spacing` is equal to `8px` right now.  Spacing is a theme value that can be changed as well but we probably don't want to edit it ourselves.

**Comment by brennancheung** [comment]
Also, please include before and after screenshots in the PR for theme changes.

**Comment by kevinylee1991** [comment]
Screen Shot 2019-12-03 at 12 27 12 PM
Screen Shot 2019-12-03 at 12 24 32 PM

File: src/app/core/components/widgets/WidgetCard.js [hunk 3]
```diff
@@ -9,11 +9,14 @@ const useStyles = makeStyles(theme => ({
...
   header: {
     padding: theme.spacing(1.5, 2),
-    backgroundColor: 'rgba(74, 163, 223, .2)',
+    backgroundColor: 'rgba(74, 163, 223, 0.1)',
   },
   content: {
     display: 'flex',
     justifyContent: 'center',
+    backgroundColor: 'rgba(74, 163, 223, 0.1)',
+    paddingTop: theme.spacing(0),
```
Comment (brennancheung) [inline src/app/core/components/widgets/WidgetCard.js]: I'd just put `0` hard-coded in this case.

File: src/app/core/components/widgets/WidgetCard.js [hunk 4]
```diff
@@ -9,11 +9,14 @@ const useStyles = makeStyles(theme => ({
   },
   header: {
     padding: theme.spacing(1.5, 2),
-    backgroundColor: 'rgba(74, 163, 223, .2)',
+    backgroundColor: 'rgba(74, 163, 223, 0.1)',
```
Comment (brennancheung) [inline src/app/core/components/widgets/WidgetCard.js]: These values should come from theme.

**Comment by kevinylee1991** [comment]
Screen Shot 2019-12-04 at 10 36 30 AM
Screen Shot 2019-12-04 at 10 36 47 AM
Screen Shot 2019-12-04 at 10 42 49 AM
Screen Shot 2019-12-04 at 10 43 00 AM

File: src/app/core/components/widgets/WidgetCard.js [hunk 5]
```diff
@@ -9,11 +9,14 @@ const useStyles = makeStyles(theme => ({
   },
   header: {
     padding: theme.spacing(1.5, 2),
-    backgroundColor: 'rgba(74, 163, 223, .2)',
+    backgroundColor: theme.palette.common.teal10,
```
Comment (brennancheung) [inline src/app/core/components/widgets/WidgetCard.js]: Name the palette keys by their key or what they represent, not what color they actually are.  It won't make sense during customization to change `teal10` to be another color.

File: src/app/core/components/widgets/WidgetCard.js [hunk 6]
```diff
@@ -9,11 +9,14 @@ const useStyles = makeStyles(theme => ({
...
   },
   content: {
     display: 'flex',
     justifyContent: 'center',
+    backgroundColor: theme.palette.common.teal10,
+    paddingTop: 0,
+    paddingBottom: [theme.spacing(1.5), '!important'],
```
Comment (brennancheung) [inline src/app/core/components/widgets/WidgetCard.js]: `!important` shouldn't be required. It's probably a sign we are doing things wrong or modifying the wrong style.

**Pull request merged**